package.json

```diff
@@ -24,18 +24,20 @@
   },
   "homepage": "https://github.com/AmpersandJS/ampersand-paginated-subcollection",
   "devDependencies": {
-    "ampersand-collection": "^1.4.0",
+    "ampersand-collection": "^1.4.5",
     "jshint": "^2.6.0",
-    "precommit-hook": "^1.0.7",
+    "precommit-hook": "^2.0.1",
     "run-browser": "^2.0.1",
-    "tap-spec": "^2.2.0",
+    "tap-spec": "^3.0.0",
     "tape": "^3.4.0"
   },
+  "files": [
+    "ampersand-paginated-subcollection.js"
```

Comment:
I had the same question when we started doing this.  Turns outreadme  is included by default (this is documented *somewhere* but I can't find it right now), we've been doing this in most of the other repos